web/src/api/index.ts

```diff
@@ -23,6 +53,24 @@ function delay(amount: number): Promise<void> {
   return new Promise((resolve) => setTimeout(resolve, amount));
 }
 
+export async function logIn(
+  username: string,
+  password: string,
+): Promise<string | undefined> {
+  const response = await client.post('/login', {
+    headers: { Authorization: 'Basic ' + btoa(username + ':' + password) },
+  });
+  setCurrentSessionToken(response.data);
+  if (!response.response.ok) {
+    return response.error;
```

Comment:
This is fine for now, but we should have a discussion about error handling in the frontend and how we display these to the user.

Reply:
Yep definitely.